Automattic/newspack-plugin
# Add Color Picker component

### All Submissions:






### Changes proposed in this Pull Request:

This PR introduces a new Newspack component: `ColorPicker`

It's based on a combination of `@wordpress/components`'s `ColorPalette` and `ColorIndicator` 

![color-picker](https://user-images.githubusercontent.com/177929/74047276-fa9af400-49c7-11ea-8110-cf6ee30d411c.png)

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Check the Components demo

### Other information:
